src/transformers/utils/hub.py

```diff
@@ -1081,6 +1093,27 @@ def extract_info_from_url(url):
     return {"repo": cache_repo, "revision": revision, "filename": filename}
 
 
+def create_and_tag_model_card(repo_id, tags=None):
+    """
+    Creates a dummy model card and tags it.
+    """
+    model_card = None
+
```

Comment:
```suggestion
```
(is it necessary?)

Reply:
True yeah this is not necessary indeed !